### DISCUSSION TITLE
Добро пожаловать на обсуждение благотворительности банды!!!

### DISCUSSION THREAD

**Discussion by DanilaDemakhin** [DemFirstNewOrg/Charity#1] [Announcements]
**ОБСУЖДАЕМ**
Всем привет на связи глава банды, я предлагаю здесь общаться и дискуссировать всё что можно по поводу нашей так называемой благотворительности в банде, йоу 😄 
 Предлагаю такую тему
https://www.tooplate.com/view/2107-new-spot

**Comment by LikOchka** [comment]
Я ХОЧУ ШОБ МЫ НЕ ЗАНИМАЛИСЬ БЛАГОТВОРИТЕЛЬНОСТЬЮ, НАМ ДЕНЬГИ НУЖНЕЕ!!!! 😠

**Comment by LikOchka** [comment]
МНЕ НРАВ САЙТИК
